gulpfile.js

```diff
@@ -13,3 +14,16 @@ gulp.task('travis-ci', ['test', 'eslint']);
 		.pipe(chug({ tasks: [task] }))
 	)
 });
+
+// all of the subdirectories in packages get installed by lerna, but for
+// various reasons, the integration tests are not under packages, so we need
+// to call npm install in that dir as part of the build.
+gulp.task('install-integration-test', () => gulp.src(['./integration-tests/package.json'])
+  .pipe(install())
+);
+
+// run the integration tests.
+gulp.task('integration-test', ['install-integration-test'], () => gulp
+	.src(['./integration-tests/gulpfile.babel.js'], {read: false})
+	.pipe(chug({tasks: ['test']}))
+);
```

Comment:
This looks good.  I have a feeling we'll want to generalize it pretty soon.  We have other packages that don't really make sense to publish (e.g. `react-server-test-pages`).